infrastructure/k8s/bin/helmit.py

```diff
@@ -75,13 +80,28 @@ def render_cronjob(project_name, secret_keys, deployment):
     context.update(deployment)
     format_values(context)
     template = yaml.load(open("templates/cron-job.yaml"), Loader=yaml.SafeLoader)
-    print("---")
-    print(yaml.dump(jsone.render(template, context)))
-
+    # print("---")
+    # print(yaml.dump(jsone.render(template, context)))
+    write_file(template, context, "cron")
+
+def write_file(template, context, prefix):
+    filename = chartsdir + "/taskcluster-" + prefix + "-" + p.split("/")[-1:][0]
```

Comment:
I'm not sure exactly what you're going for here, since `p` is undefined.

Here's an idea I think would work:

For all charts, lets make the name be a combination of the `project_name` and the json-e file name (which matches the k8s type). For deployments, let's tack on the `proc_name`, and for cronjobs the `job_name` since there can be more than one of those things per service.

So e.g. for the purge-cache service we would have

* taskcluster-purge-cache-role.yaml
* taskcluster-purge-cache-role-binding.yaml
* taskcluster-purge-cache-service-account.yaml
*  taskcluster-purge-cache-deployment-web.yaml
* taskcluster-purge-cache-cron-job-expireCachePurges.yaml

Reply:
the rbacs split into "role", "role-binding", "service-account"